specification/billing/resource-manager/Microsoft.Billing/stable/2021-10-01/payment.json

```diff
@@ -3,7 +3,7 @@
   "info": {
     "version": "2021-10-01",
     "title": "PaymentManagementClient",
-    "description": "Payment management client provides access to payment resources for Azure."
+    "description": "Billing client provides access to billing resources and logs for Azure subscriptions."
```

Comment:
Now we have more than Azure in Billing RP can we make this generic:

"The billing client allows you to view and manage your billing details programmatically."